src/plone/restapi/serializer/collection.py

```diff
@@ -14,16 +16,19 @@ class SerializeCollectionToJson(SerializeToJson):
 
     def __call__(self):
         result = super(SerializeCollectionToJson, self).__call__()
-        portal = getSite()
-        result['member'] = [
-            {
-                '@id': '{0}/{1}'.format(
-                    portal.absolute_url(),
-                    '/'.join(member.getPhysicalPath())
-                ),
-                'title': member.title,
-                'description': member.description
-            }
-            for member in self.context.results()
-        ]
+
+        # XXX: collection.results() claims to return an IContentListing
```

Comment:
Ist this still true? Looking at the `ContentListingCollection` view implementation in Plone 5, it simply calls `self.context.results()`. Thus calling the view or calling `results()` directly with the same parameters should give the same result.